tensorflow_recommenders_addons/dynamic_embedding/core/kernels/hkv_hashtable_op_gpu.cu.cc

```diff
@@ -81,9 +83,17 @@ class HkvHashTableOfTensorsGpu final : public LookupInterface {
         ctx, (max_hbm_for_vectors_i64 >= 0),
         errors::InvalidArgument("params max_hbm_for_vectors less than 0"));
 
+    OP_REQUIRES_OK(ctx, GetNodeAttr(kernel->def(), "evict_global_epoch",
```

Comment:
The `evict_global_epoch` is required only when the strategy is `EpochLru` or `EpochLfu`.

Reply:
The default is 0. It doesn't matter, does it?